components/PaymentPaypal.vue

```diff
@@ -19,11 +19,9 @@ import PaypalButton from '../components/Button'
 
 export default {
   name: 'PaymentPaypal',
-  props: {
-    platformTotals: {
-      type: Object,
-      required: true,
-      default: () => {}
+  computed: {
+    grandTotal () {
+      return this.$store.state.cart.platformTotals['grand_total'] || 0
```

Comment:
For dynamically created components no access to `this.$store`.

Reply:
In PR #39 I've added an additional parameter which defines the parent instance of the dynamic component. Due to this change `this.$store` should be accessible...

```javascript
      const componentInstance = (new Component({
        propsData: { platformTotals: app.$store.state.cart.platformTotals },
        parent: app
      }))
```

Reply:
Ok, good.